CHANGELOG.md

```diff
@@ -187,7 +191,7 @@ New features:
 
 Bugfixes:
 - Don't overwrite files when doing `init`, just skip the copy if some file exists (#56)
-- Print `git` output in case of failure when doing `install` (#54, #59) 
+- Print `git` output in case of failure when doing `install` (#54, #59)
```

Comment:
Not too happy with the diff here, apparently my editor removed the whitespaces, other than that it should be the same. I'll remove if you'd like.

Reply:
No worries, this is fine 🙂